rllib/models/tf/tf_distributions.py

```diff
@@ -105,8 +105,8 @@ def __init__(
 
         if logits is not None:
             assert temperature > 0.0, "Categorical `temperature` must be > 0.0!"
-            _logits = logits / temperature
-            probs = tf.nn.softmax(_logits, axis=-1)
+            logits /= temperature
```

Comment:
This is now analogous to the old tf Categorical distribution class and seems to add stability to test cases and Atari benchmarks (without this change, sometimes, the experiment crashes with actions being sampled outside the action space).

Reply:
you mean using tf.nn.sparse_softmax_cross_entropy_with_logits vs. self._dist.log_prob(value, **kwargs)

Reply:
Yeah, sorry, the whole code block was adjusted to match the established logic.
This might actually fix the broken PPO+RLM test case ...

Reply:
ooo I see. tf magic :)